### PR TITLE
[pm-10995] feature flag removal

### PR DESCRIPTION
**Pull request by cd-bitwarden**
## 📔 Objective
Remove feature flag vault-bulk-management-action

Renaming the branch, so that tests pass and it made me create a new PR 

https://github.com/bitwarden/clients/pull/10784

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/web/src/app/vault/individual-vault/vault.component.ts [hunk 1]
```diff
@@ -432,9 +426,6 @@ export class VaultComponent implements OnInit, OnDestroy {
         case "viewAttachments":
           await this.editCipherAttachments(event.item);
           break;
-        case "viewCipherCollections":
-          await this.editCipherCollections(event.item);
```
Comment (gbubemismith) [inline apps/web/src/app/vault/individual-vault/vault.component.ts:436]: ⛏️ I think we can remove the `editCipherCollections` function

**Review by gbubemismith — CHANGES_REQUESTED** [review]
I left some tiny nits; other than that, this looks good

File: apps/web/src/app/vault/individual-vault/vault.component.ts (hunk 1 above)
Comment (cd-bitwarden) [reply apps/web/src/app/vault/individual-vault/vault.component.ts:436]: You're right 👍

**Review by gbubemismith — APPROVED** [review]

**Review by shane-melton — DISMISSED** [review]

File: apps/web/src/app/vault/components/vault-items/vault-items.component.html [hunk 2]
```diff
@@ -60,21 +60,12 @@
...
-              <i class="bwi bwi-fw bwi-arrow-circle-right" aria-hidden="true"></i>
-              {{ "moveSelectedToOrg" | i18n }}
-            </button>
             <button *ngIf="showBulkTrashOptions" type="button" bitMenuItem (click)="bulkRestore()">
               <i class="bwi bwi-fw bwi-undo" aria-hidden="true"></i>
               {{ "restoreSelected" | i18n }}
             </button>
             <button
-              *ngIf="deleteAllowed || showBulkTrashOptions"
+              *ngIf="showDelete || showBulkTrashOptions"
```
Comment (gbubemismith) [inline apps/web/src/app/vault/components/vault-items/vault-items.component.html:68]: :warning: I missed this. `showDelete` won't execute as it's not a getter

**Review by gbubemismith — CHANGES_REQUESTED** [review]
I noticed this while trying to align my PR with yours

File: apps/web/src/app/vault/individual-vault/vault.component.ts [hunk 3]
```diff
@@ -215,7 +201,6 @@ export class VaultComponent implements OnInit, OnDestroy {
     private eventCollectionService: EventCollectionService,
     private searchService: SearchService,
     private searchPipe: SearchPipe,
-    private configService: ConfigService,
```
Comment (shane-melton) [inline apps/web/src/app/vault/individual-vault/vault.component.ts:218]: ⚠️ The `configService` can no longer be removed as its being used to get the `extension-refresh` flag below.

**Review by shane-melton — CHANGES_REQUESTED** [review]

File: apps/web/src/app/vault/individual-vault/vault.component.ts (hunk 3 above)
Comment (cd-bitwarden) [reply apps/web/src/app/vault/individual-vault/vault.component.ts:218]: Thank you for noticing this Shane!

File: apps/web/src/app/vault/org-vault/vault.component.ts [hunk 4]
```diff
@@ -626,9 +626,6 @@ export class VaultComponent implements OnInit, OnDestroy {
         case "viewAttachments":
           await this.editCipherAttachments(event.item);
           break;
-        case "viewCipherCollections":
-          await this.editCipherCollections(event.item);
```
[the remark this answers is not in the document]
Comment (gbubemismith) [reply apps/web/src/app/vault/org-vault/vault.component.ts:615]: 🤔 I think we should leave `viewCipherCollections` for now, as AC doesn't use `assign to collections` yet

**Review by shane-melton — APPROVED** [review]

**Review by gbubemismith — APPROVED** [review]

**Pull request merged**